xchange-kucoin/src/main/java/org/knowm/xchange/kucoin/KucoinAdapters.java

```diff
@@ -235,7 +235,7 @@ public static OrderBook adaptOrderBook(CurrencyPair currencyPair, OrderBookRespo
             .sorted(Ordering.natural().onResultOf((PriceAndSize s) -> s.price).reversed())
             .map(s -> adaptLimitOrder(currencyPair, BID, s, timestamp))
             .collect(toCollection(LinkedList::new));
-    return new OrderBook(timestamp, asks, bids);
+    return new OrderBook(timestamp, asks, bids, true);
```

Comment:
I think by passing true here, you're overriding the sorting done above, because then OrderBook constructor resorts the asks and bids again